<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student</class>
 <widget class="QDialog" name="Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>482</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>502</width>
    <height>482</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>341</width>
         <height>361</height>
        </size>
       </property>
       <property name="title">
        <string>学生信息填写</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="ID_label">
            <property name="text">
             <string>  学号   </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ID_lineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="name_label">
            <property name="text">
             <string>  姓名   </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="name_lineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="sex_label">
            <property name="text">
             <string>  性别  </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QFrame" name="sex_frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QRadioButton" name="sex_man_radioButton">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>0</y>
               <width>61</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>男</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="sex_woman_radioButton">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>0</y>
               <width>61</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>女</string>
             </property>
            </widget>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="grade_label">
            <property name="text">
             <string>  年级  </string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="grade_lineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="major_label">
            <property name="text">
             <string>  专业  </string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="major_lineEdit"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="introduce_label">
            <property name="text">
             <string>  社团  </string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QTextEdit" name="introduce_textEdit"/>
          </item>
          <item row="6" column="0">
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>  简介  </string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPlainTextEdit" name="plainTextEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>98</width>
         <height>341</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="accept_pushButton">
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancel_pushButton">
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reset_pushButton">
          <property name="text">
           <string>重置</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
